src/content/en/tools/chrome-devtools/beginners/css.md

```diff
@@ -2,82 +2,114 @@ project_path: /web/tools/_project.yaml
+tutorial. [Set up your code](#setup) will show you how to get set up.
 
 Currently your site looks like this: 
 
-`(TODO: A screenshot of the basic HTML site)`
+<figure>
+  <img src="imgs/css/intro1.png"
+       alt="What your site currently looks like."/>
+  <figcaption>
+    <b>Figure X</b>. What your site currently looks like
+  </figcaption>
+</figure>
 
 After completing the tutorial, it will look like this: 
 
-`(TODO: A screenshot of the basic CSS site)`
-
+<figure>
+  <img src="imgs/css/intro2.png"
+       alt="What your site will look like at the end of the tutorial."/>
+  <figcaption>
+    <b>Figure X</b>. What your site will look like at the end of the tutorial
+  </figcaption>
+</figure>
 
 ## Goals {: #goals }
 
 By the end of this tutorial, you will understand:
 
-* How CSS can be used to style webpages.
-* How Chrome DevTools can help you when you're working CSS.
-* The difference between CSS and CSS Frameworks.
+* How to use CSS to style a web page.
+* How to use Chrome DevTools to experiment with CSS.
+* The difference between CSS and CSS frameworks.
 
 You'll also have a real website!
 
 ## Prerequisites {: #prerequisites }
 
 Before attempting this tutorial, complete the following prerequisites:
 
-* Complete the previous tutorial on HTML and DOM or complete *Setup* below.
-* Download the [Google Chrome][chrome]{: .external } web browser. This tutorial uses a set of web development tools, 
-called Chrome DevTools, that are built into Google Chrome. 
+* Comlete [Get Started with HTML and the DOM][HTML] or make sure that you have an understanding
+  of HTML and the DOM similar to what's taught in that tutorial.
+* Download the [Google Chrome][chrome]{: .external } web browser. This tutorial uses a set of
+  web development tools, called Chrome DevTools, that are built into Google Chrome. 
+
 [chrome]: https://www.google.com/chrome/
 
 ## Set up your code {: #setup }
 
- 
-
 In order to start creating your site, you need to set up your code:
 
 1. **If you have already completed the first tutorial in this series, skip this section! 
    Continue using your code from the last tutorial, 
    [Getting Started with HTML and the DOM](/web/tools/chrome-devtools/beginners/html).**
 2. Open the [source code](https://glitch.com/edit/#!/dfb2). A code editor called 
-   Glitch shows a page called `index.html`. This will be called the editing tab.
+   Glitch shows a page called `index.html`. This tab of your browser will be called the
+   **editing tab**.
 
-`(TODO: A screenshot showing the action.)`
+     <figure>
+       <img src="imgs/css/setup1.png"
+            alt="The source code."/>
+       <figcaption>
+         <b>Figure X</b>. The source code
+       </figcaption>
+     </figure>
 
 2. Click **dfb2**. A menu pops up.
 
-`(TODO: A screenshot showing the menu)`
+     <figure>
+       <img src="imgs/css/setup2.png"
+            alt="The editor tab."/>
+       <figcaption>
+         <b>Figure X</b>. The editor tab
```

Comment:
Use the placeholder `Figure X` instead of trying to number the figures. Numbering the figures is one of the last things we do before publishing.